examples/settings.py

```diff
@@ -89,7 +89,7 @@ def make_cfg(settings):
     sensor_specs = []
     for sensor_uuid, sensor_params in sensors.items():
         if settings[sensor_uuid]:
-            sensor_spec = habitat_sim.SensorSpec()
+            sensor_spec = habitat_sim.CameraSensorSpec()
```

Comment:
Before, we only have one type of sensor specification and you can do it like this in a simple way.
Now, we have different sensor specifications based on sensor type and sensor sub-type, so changes are needed accordingly. Cannot just construct the camera sensor by default.

Which specific sensor spec should be used? 
Check the sensor type first, if it is visual sensor, then check the sensor sub-type, if it is "Pinhole" or "Ortho", then select the `CameraSensorSpec()`